src/auth/AuthClientHandler.h

```diff
@@ -33,19 +32,16 @@ class AuthClientHandler {
   uint32_t want;
   uint32_t have;
   uint32_t need;
-  mutable SharedMutexT<lock_policy> lock;
```

Comment:
AuthClientHandler no longer needs to be templated, only AuthClientHandler::create() does. that can get rid of the rest of the `using AuthClientHandler<lock_policy>::` stuff in subclasses